sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,366 @@
+    "\n",
+    "**Learning Objectives** - By the end of this tutorial, you should be able to:\n",
+    "- Create Azure Machine Learning workspace with Workspace Managed Network from Python SDK\n",
+    "- Get the details of a workspace from Python SDK\n",
+    "- Understand provisioning the Managed Network manually\n",
+    "- Manage the Isolation Mode of the Managed Network\n",
+    "- Configure User Defined Outbound Rules for the Managed Network\n",
+    "\n",
+    "**Motivations** - The [workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace) is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. Ability to create and manage workspace is a prerequisite for any activity in Azure Machine Learning.\n",
+    "\n",
+    "**Reference** - This example will act as a starting point to get going with the [Managed Network](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-managed-network) capabilities of AzureML workspaces. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Create a basic workspace\n",
+    "\n",
+    "## 1.1. Import the required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries for this notebook\n",
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.entities import (\n",
+    "    Workspace,\n",
+    "    ManagedNetwork,\n",
+    "    IsolationMode,\n",
+    "    FqdnDestination,\n",
+    "    ServiceTagDestination,\n",
+    "    PrivateEndpointDestination,\n",
+    ")\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "import datetime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2 Configure where workspace needs to be created.\n",
+    "Before creating a workspace, we need identifier parameters - a subscription and resource group. We will use these parameters to define where the Azure Machine Learning workspace will be created.\n",
+    "\n",
+    "The `MLClient` from `azure.ml` will be used to create the workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. reference the [configuration notebook](../../jobs/configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "subscription_id"
+   },
+   "outputs": [],
+   "source": [
+    "# Enter details of your subscription\n",
+    "subscription_id = \"0caf7ec9-615a-4491-bad8-64ce023324e1\"\n",
```

Comment:
change this for 0s

Reply:
it was leftover from my testing. it should be 

subscription_id = "<SUBSCRIPTION_ID>"
resource_group = "<RESOURCE_GROUP>"

since its run against CI